changelogs/fragments/2008-update-java-cert-replace-cert-when-changed.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
+  - "Java_cert: Change `state: present` to check certificates by hash, not just alias name (https://github.com/ansible/ansible/issues/43249)"
+  - "Java_cert: cosmetic - modify run_command indentations"
```

Comment:
```suggestion
```
I don't think any user of the module cares if indentations in its code changed :)